### PR TITLE
Dark mode

### LINKED ISSUE #736: Add Dark Mode Support
**Describe the feature you'd like**

Users should be able to switch themes to dark mode. 

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by prathamk22**
Fixes #736
Changes: 
Changed all instances of black to head
Changed all instances of greyish to subHead
Made new night-color, night-style file for light and dark theme
Since it is a draft pull request, some changes are left.
Screenshots for the change:
https://youtu.be/fDXmWAcF2fM

### CODE REVIEW

File: app/src/main/res/values-night/styles.xml [hunk 1]
```diff
@@ -0,0 +1,282 @@
+<resources>
```
Comment (aggarwalpulkit596) [inline app/src/main/res/values-night/styles.xml:1]: Why do we need a style file for dark mode?

File: app/src/main/res/values-night/colors.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <color name="battleship_gray">#64707b</color>
+    <color name="green">#8dbf75</color>
+    <color name="algae_green">#25d366</color>
+
+    <color name="dark_grey">#222222</color>
+    <color name="neon_red">#ff003a</color>
+    <color name="light_transparent">#16000000</color>
+
+    <!--    New Colors-->
+    <color name="pastel_red">#e74e4b</color>
```
Comment (aggarwalpulkit596) [inline app/src/main/res/values-night/colors.xml]: and if colors are the same for both the mode then, in that case, they shouldn't be declared again and again

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

**Comment by prathamk22** [comment]
@aggarwalpulkit596

**Comment by prathamk22** [comment]
@aggarwalpulkit596

**Comment by prathamk22** [comment]
@aggarwalpulkit596 Bhaiye isse bhi review kardo.

**Pull request closed without merging**